{ 'stand_up_front_300': { 'head': [],
                          'left_arm': [ { 'duration': 1.0,
                                          'name': u'stand_up_front_0900',
                                          'positions': [ -2.2143,
                                                         -3.1294,
                                                         -0.0006,
                                                         3.1407,
                                                         -1.8699,
                                                         1.0022,
                                                         -0.6996],
                                          'starttime': 0.0},
                                        { 'duration': 3.99,
                                          'name': u'stand_up_front_1000',
                                          'positions': [ -1.0183,
                                                         -3.1562,
                                                         -0.0659,
                                                         2.9864,
                                                         -1.7841,
                                                         1.0408,
                                                         0.2877],
                                          'starttime': 1.0}],
                          'left_leg': [ { 'duration': 3.002,
                                          'name': u'stand_up_front_1000',
                                          'positions': [ -0.0326,
                                                         -0.0984,
                                                         -1.5522,
                                                         2.6052,
                                                         1.8114,
                                                         -0.0906],
                                          'starttime': 0.999}],
                          'right_arm': [ { 'duration': 0.999,
                                           'name': u'stand_up_front_0900',
                                           'positions': [ 2.2143,
                                                          3.1294,
                                                          0.0006,
                                                          -3.1407,
                                                          1.8699,
                                                          -1.0022,
                                                          0.6996],
                                           'starttime': 0.0},
                                         { 'duration': 4.002,
                                           'name': u'stand_up_front_1000',
                                           'positions': [ 1.0183,
                                                          3.1562,
                                                          0.0659,
                                                          -2.9864,
                                                          1.7841,
                                                          -1.0408,
                                                          -0.2877],
                                           'starttime': 0.999}],
                          'right_leg': [ { 'duration': 2.991,
                                           'name': u'stand_up_front_1000',
                                           'positions': [ 0.0326,
                                                          0.0984,
                                                          1.5522,
                                                          -2.6052,
                                                          -1.8114,
                                                          0.0906],
                                           'starttime': 1.014}],
                          'torso': []}}
